src/Core/tests/gtest_DecimalFunctions.cpp

```diff
@@ -144,6 +146,17 @@ TEST_P(DecimalUtilsSplitAndCombineForDateTime64Test, getFractionalPartDateTime64
 
 }
 
+namespace std
```

Comment:
There is one report from `clang-tidy`

```
Oct 22 20:03:51 /build/src/Core/tests/gtest_DecimalFunctions.cpp:152:16: error: modification of 'std' namespace can result in undefined behavior [cert-dcl58-cpp,-warnings-as-errors]
```

Which should be fixed by the following patch:

```suggestion
namespace std // NOLINT(cert-dcl58-cpp)
```